.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+name: Build Pipeline
+on:
+  push:
+    branches:
+      - main
+jobs:
+  FLHook:
+    runs-on: windows-latest
+    if: ${{ !contains(github.event.head_commit.message, 'Updated FLHook') }}
```

Comment:
I personally prefer writing commit messages using the present simple tense (e.g. "Update FLHook").